package.json

```diff
@@ -39,5 +39,8 @@
   "resolutions": {
     "react-error-overlay": "6.0.9"
   },
-  "packageManager": "yarn@3.1.0"
+  "packageManager": "yarn@3.1.0",
+  "dependencies": {
```

Comment:
Propose to move this dependency to `ketcher-react`

Reply:
Fixed